tap_google_sheets/client.py

```diff
@@ -145,6 +146,16 @@ def __init__(self,
         self.__expires = None
         self.__session = requests.Session()
         self.base_url = None
+        self.supports_all_drives = False
+
+        # if supports_all_drives is given as invalid value then raise exception
+        if supports_all_drives not in ('true', 'True', True, 'false', 'False', False, None):
+            raise Exception("Invalid Parameter value: The given value of the supports_all_drives is an invalid value.")
+
+         # if supports_all_drives is given as true then use supports_all_drives
+        if supports_all_drives in ('true', 'True', True):
+            self.supports_all_drives = True
+
```

Comment:
```suggestion
        self.supports_all_drives = supports_all_drives in ('true', 'True', True)
```

Reply:
Hi @cosimon, Line 158 to 160 are raising exceptions for invalid values for configuring parameter supports_all_drives, so we can not remove it, but we made other changes per your suggestion.